### ISSUE TITLE
100 seconds is sometimes not enough timeout

### ISSUE THREAD

**Issue by athornton** [PDP-10/its#1649]
I'm building on a Raspberry Pi 3B+.

The 100 second timeout during disk creation is insufficient although 300s seems to work OK.  It should either be set higher or it should do something to test the environment and determine an appropriate value.

**Comment by larsbrinkhoff** [comment]
I have a work in progress to increase the time limits where necessary: #1613.

**Comment by larsbrinkhoff** [comment]
With your input in mind, maybe it would be best to just set the baseline timeout to 300.

**Comment by eswenson1** [comment]
I had to do this

**Comment by larsbrinkhoff** [comment]
@eswenson1, I guess you didn't mean to close this?

**Comment by eswenson1** [comment]
No, I didn’t. Sorry. And I tried to delete the comment — which is why it is missing.

**Comment by larsbrinkhoff** [comment]
@PiDP also wrote in to report his rPi 3 can't finish the build.  If timeout 300 works, maybe that's the way to go.

I believe running off an SD card is a major part of rPi low performance.  I have an idea that we can test disk speed and warn the user of if it's slow.

**Comment by athornton** [comment]
Yup, this bit me again yesterday.  I was lazy and just added another zero to the end of that 100 and it worked fine.
